source/hosts-resolver.js

```diff
@@ -52,14 +51,13 @@ class HostsResolver {
 	async _update() {
 		try {
 			let lines = await readFile(this._hostsPath, fileOptions);
-			lines = lines.replace(whitespaceRegExp, ' ');
-			lines = lines.replace(tabRegExp, ' ');
-			lines = lines.replace(startsWithWhitespaceRegExp, '');
```

Comment:
I wanted to avoid calling a RegExp every line. While I had good intentions, it turned the other way. Calling `line.replace(whitespaceRegExp, ' ').trim()` is the easiest way.